translations/health/cy/cris_0001.po

```diff
@@ -668,8 +668,8 @@ msgid "Why could you not successfully take your swab sample?"
 msgstr "Pam nad oedd modd i chi gymryd eich sampl swab yn llwyddiannus?"
 
 #. Question text
-msgid "Is this your swab sample barcode {swab_test_barcode} ?"
-msgstr "Ai cod bar eich sampl swab yw {swab_test_barcode} ?"
+msgid "Is this your swab sample barcode {first_name} ?"
```

Comment:
The content doesn't make sense but that is on purpose, these question won't actually be shown to users at the moment. This is a work around as we don't have the barcodes in scope.